libs/libpam/Makefile

```diff
@@ -33,23 +38,22 @@ define Package/libpam/description
 	The Linux-PAM Pluggable Authentication Modules.
 endef
 
-TARGET_CFLAGS += $(FPIC)
```

Comment:
Sure.

In most packages that I've seen here, FPIC is used when not doing so would break the build. 

I also see that -fpic is being passed by libpam's own build system.

To mandate PIE/PIC, there's a PKG_ASLR_PIE variable as well.

In short, I don't think it's necessary anymore.

Reply:
The configure script also has a --with-pic parameter.